### ISSUE TITLE
Old avatar pictures not updating in Timeline

### ISSUE THREAD

**Issue by xpkoala** [Crypton/statusapp#155]
Not sure if this is working as intended or not. If you change your avatar on the contact card any entry you have made prior to the change will stay as the old avatar picture.

>-Post to timeline at time X - (Avatar picture = pic1)
>-Change Avatar to pic2
>-Post to timeline at time Y 
>-Look back to post made at time X; picture on display is pic1

**Comment by daviddahl** [comment]
That is correct behavior for now. We should update all of them, but right now it just looks for empty avatars (with first initial) or updates the new posts with the latest avatar
